lib/utils/validations.js

```diff
@@ -1,3 +1,38 @@
+import Joi from 'joi'
+import { payloadSchema } from './schema'
+import getEntityName from 'contentful-batch-libs/dist/utils/get-entity-name'
+
+const attachEntityName = (details, payload) => {
+  details.map((detail) => {
+    if (detail.path.length >= 2) {
+      detail.entity = getEntityName(payload[detail.path[0]][detail.path[1]])
+    }
+    return detail
+  })
+}
+const countInvalidEntities = (validationData) => {
+  const entityCount = validationData.reduce((entities, currentDetail) => {
+    if (!entities[currentDetail.path[0]]) {
+      entities[currentDetail.path[0]] = 1
+    } else {
+      entities[currentDetail.path[0]]++
+    }
+    return entities
+  }, {})
+  return Object.keys(entityCount).map((key) => `${key}:${entityCount[key]}`)
```

Comment:
What about if we pass key through https://lodash.com/docs/4.17.4#startCase. IMHO that would improve readability :)

(We probably could do this then later on on more parts in the code to properly format entity types)

Reply:
my idea is to map this key to the key in the JSON payload. I think it is fine like that